src/frontend/components/DashboardConfirmButton/index.spec.tsx

```diff
@@ -0,0 +1,129 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import fetchMock from 'fetch-mock';
+import { Grommet } from 'grommet';
+import { wrapInIntlProvider } from '../../utils/tests/intl';
+import { DashboardConfirmButton } from '.';
+
+const props = {
+  confirmationLabel: <small>Are you sure ?</small>,
+  label: <span>Start a live</span>,
+  onConfirm: jest.fn(),
+};
+
+jest.mock('../../data/appData', () => ({
+  appData: {
+    document: null,
+    jwt: 'cool_token_m8',
+  },
+}));
```

Comment:
It seems like appData isn't used by DashboardConfirmButton or ConfirmationLayer.
Can we remove this ?

Reply:
nope, otherwise test don't go through

Reply:
ok :+1: